app.py

```diff
@@ -119,40 +119,42 @@ def plant_conditions(scientific_name_raw: str):
                            'FROM PlantIllness '
                            'WHERE plantID = '
                            '    (SELECT id FROM Plant WHERE scientificName = %s);', (scientific_name,))
-            conditions = [dict(zip(response_keys, condition)) for condition in cursor.fetchall()]
+            illnesses = [dict(zip(response_keys, condition)) for condition in cursor.fetchall()]
     conn.close()
     return {
-        'conditions': conditions
+        'illnesses': illnesses
     }
 
-@app.route('/plants/<scientific_name_raw>/symptoms', methods=['GET'])
-def plant_symptoms(scientific_name_raw: str):
+@app.route('/plants/<illness_name_raw>/symptoms', methods=['GET'])
```

Comment:
Can we make this `/plants/<scientific_name_raw>/illness/<illness_name_raw>/symptoms` or something? It's a bit long, but having the parameter in interactions 4 and 5 mean different things depending on what comes after it is a bit confusing.